ui/helpers/utils/util.test.js

```diff
@@ -301,4 +301,31 @@ describe('util', () => {
       });
     });
   });
+
+  describe('toHumanReadableTime()', () => {
+    const t = (key, number) => {
+      switch (key) {
+        case 'gasTimingSecondsShort':
+          return `${number} sec`;
+        case 'gasTimingMinutesShort':
+          return `${number} min`;
+        case 'gasTimingHoursShort':
+          return `${number} hrs`;
+        default:
+          return '';
+      }
+    };
+    it('should return empty string if milliseconds passed is undefined', () => {
+      expect(util.toHumanReadableTime(t)).toStrictEqual('');
+    });
+    it('should return value in seconds for milliseconds passed is less than 9000, that is 90 seconds', () => {
+      expect(util.toHumanReadableTime(t, 6000)).toStrictEqual('6 sec');
+    });
+    it('should return value in minutes for milliseconds passed is greater than 90000', () => {
```

Comment:
it shows minutes only after 90000 milliseconds

Reply:
so its 
<= 90000 - sec
<= 5400000 - min
else hrs

Reply:
Okay, that confirms what I thought was going on. Do you mind writing a test for that case in which it's >= 60000 but <= 90000? And same for below. Sometimes it's nice to be able to read the tests to better understand the underlying logic rather than look at the code so I think this could do a nice job of illustrating that.

Reply:
👍 I added more test cases